cli/state.go

```diff
@@ -1505,16 +1505,24 @@ var StateSectorCmd = &cli.Command{
 		if si.SectorKeyCID != nil {
 			fmt.Println("SectorKeyCID: ", si.SectorKeyCID)
 		}
-		fmt.Println("DealIDs: ", si.DealIDs)
+		fmt.Println("DealIDs (deprecated): ", si.DeprecatedDealIDs)
 		fmt.Println()
 		fmt.Println("Activation: ", cliutil.EpochTimeTs(ts.Height(), si.Activation, ts))
 		fmt.Println("Expiration: ", cliutil.EpochTimeTs(ts.Height(), si.Expiration, ts))
 		fmt.Println()
 		fmt.Println("DealWeight: ", si.DealWeight)
 		fmt.Println("VerifiedDealWeight: ", si.VerifiedDealWeight)
 		fmt.Println("InitialPledge: ", types.FIL(si.InitialPledge))
-		fmt.Println("ExpectedDayReward: ", types.FIL(si.ExpectedDayReward))
-		fmt.Println("ExpectedStoragePledge: ", types.FIL(si.ExpectedStoragePledge))
+		edr := big.Zero()
+		if si.ExpectedDayReward != nil {
+			edr = *si.ExpectedDayReward
+		}
+		fmt.Println("ExpectedDayReward: ", types.FIL(edr))
+		esp := big.Zero()
+		if si.ExpectedStoragePledge != nil {
+			esp = *si.ExpectedStoragePledge
+		}
+		fmt.Println("ExpectedStoragePledge: ", types.FIL(esp))
```

Comment:
I'd consider just not printing these fields in this case.

Reply:
Ideally, we'd just ignore these fields entirely after nv25 (i.e., get the network version for the given tipset).